context-free.md

```diff
@@ -1,22 +1,30 @@
 # Context Free
 
 ##  Who is this programming language for?
+This language is for artists who are tech savvy or programming inclined. Target markets include artists that want to dabble in tech and techies that want to dabble in art.
 
 
 ## What is easy to do in this language? Why is it easy?
+Its easy to draw shapes. There is a lot of built in infrastructure to create shapes of all kinds, sizes and colors in only a few (1 to 5) lines of code. You can do recursion relatively easily and make pictures that are made up of many shapes because of the way that you are able to manipulate the x and y coordinates.
 
 
 ## What is hard to do in this language? Why is it hard?
+I found it difficult to manage the colors in context-free. It might have just been me but I found the color commands to be really finicky. In a more macro sense I found it quite difficult to draw things that are either very detailed or made up of shapes that are very curvy. The first was caused by the fact that trying to draw things on a small scale was difficult and understanding the ways in which the rules go together was confusing to me. Also the curviness of shapes that aren't built in to the Context Free grammar is difficult because it reqiures the programmer to use math to create the shapes manually.
```

Comment:
Part of the reason why understanding the rules might've be tough is that when multiple options are available in a single rule, the language picks one at random. This makes it difficult to visualize what your final diagram will look like, which is something I didn't realize until I was discussing the language with someone else.